### PR TITLE
fix(linter): Respect the "quiet" option

### LINKED ISSUE #3946: nx lint with --quiet not hiding warnings
## Current Behavior
When running `nx lint {project} --quiet`, or when adding `quiet: true` to the workspace config for the lint behaviour, I am still seeing all the warnings in the output

## Expected Behavior

when running with `quiet`, warnings should be hidden from the output logs when linting

## Steps to Reproduce

In .eslintrc.json, set up at least one rule that causes a warning (doesn't matter whether basic eslint or typescript-eslint type rule)
in project, write code that violates the warning rule
run `nx lint {project} --quiet`

### Environment

  nx : Not Found
  @nrwl/angular : 10.3.1
  @nrwl/cli : 10.3.1
  @nrwl/cypress : 10.3.1
  @nrwl/eslint-plugin-nx : 10.3.1
  @nrwl/express : Not Found
  @nrwl/jest : 10.3.1
  @nrwl/linter : 10.3.1
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 10.3.1
  @nrwl/react : 10.3.1
  @nrwl/schematics : Not Found
  @nrwl/tao : 10.3.1
  @nrwl/web : 10.3.1
  @nrwl/workspace : 10.3.1
  typescript : 3.9.7

### PR DESCRIPTION
**Pull request by arabull**
## Current Behavior
The eslint plugin doesn't properly respect the "quiet" configuration option, displaying warnings even if "quiet" is `true`.

## Expected Behavior
The eslint plugin should respect the "quiet" configuration option, not displaying errors if it is `true`.

## Related Issue(s)
Fixes #3946

### CODE REVIEW

File: packages/linter/src/builders/eslint/lint.impl.spec.ts [hunk 1]
```diff
@@ -285,6 +285,124 @@ describe('Linter Builder', () => {
...
+      await runBuilder(
+        createValidRunBuilderOptions({
+          eslintConfig: './.eslintrc.json',
+          lintFilePatterns: ['includedFile1'],
+          format: 'json',
+          silent: false,
+          quiet: true,
+        })
+      );
+      console.log('FPPP', loggerSpy.mock.calls);
```
Comment (JamesHenry) [inline packages/linter/src/builders/eslint/lint.impl.spec.ts:387]:
```suggestion

```

**Review by JamesHenry — CHANGES_REQUESTED** [review]
Extra console log left in, otherwise I think this is good to go

File: packages/linter/src/builders/eslint/lint.impl.spec.ts (hunk 1 above)
Comment (arabull) [reply packages/linter/src/builders/eslint/lint.impl.spec.ts]: Well, shoot. That was careless of me!

**Comment by ahaskell35** [comment]
should this be working with the **run-many** command as well?
current version I am using (11.2.12) with this command:
```
nx run-many --target=lint -all --quiet
``` 
does not pass the --quiet argument to the child runners.

```
nx lint {project} --quiet
```
does work as described above.

**Comment by james-wilkinson-thd** [comment]
Any idea when this till be pushed to the project? I ran a fresh install and still have this error.

**Comment by cakeinpanic** [comment]
also interested

**Comment by JamesHenry** [comment]
I just had a super quick look at a project which is not on the latest (it's on Nx `11.3.2`) and this change is present in my installed Nx, so I would say make sure you are using a version that was released after this PR was merged (11.3.2 or later for sure) and it should be there.

If you are having issues _in spite of_ this change when using the eslint executor please report them as new issues by following the issue templates

**Comment by cakeinpanic** [comment]
@JamesHenry 
here is what I'm running
```
> nx lint client --quiet
> nx run client:lint --quiet 
'quiet' is not found in schema
```

```
    "@nrwl/cli": "12.0.7",
    "@nrwl/linter": "12.0.7",
    "@nrwl/nx-cloud": "11.2.0",
    "@nrwl/workspace": "12.0.7",
```

maybe I need to update something else?

**Comment by dandean** [comment]
The `--quiet` flag doesn't appear to work via `nx affected:lint --quiet`.

```
nx --version
11.6.0
```

**Comment by JamesHenry** [comment]
@ahaskell35 @cakeinpanic @james-wilkinson-thd @dandean https://github.com/nrwl/nx/issues/5498

**Pull request merged**